tests/Unit/Domain/Creators/Test_RotatedIntervals.cpp

```diff
@@ -293,6 +324,14 @@ void test_rotated_intervals_factory() {
             {{Direction<1>::upper_xi(), {0, flipped}}}},
         std::vector<std::unordered_set<Direction<1>>>{
             {Direction<1>::lower_xi()}, {Direction<1>::lower_xi()}},
+        std::make_tuple(
+            std::pair<std::string,
+                      domain::FunctionsOfTime::PiecewisePolynomial<2>>{
+                "TranslationX",
+                {1.0, std::array<DataVector, 3>{{{0.0}, {2.3}, {0.0}}}, 10.0}}),
+        make_vector_coordinate_map_base<Frame::Grid, Frame::Inertial>(
+            CoordinateMaps::TimeDependent::Translation{"TranslationX"},
+            CoordinateMaps::TimeDependent::Translation{"TranslationX"}),
         expected_boundary_conditions);
   }
 }
```

Comment:
You have factory tests for:
No boundary condition/no time dependence
With boundary condition/with time dependence

Do you need to test for the other two combinations? (no boundary condition/with time dependence and with boundary condition/no time dependence)